### DISCUSSION TITLE
Custom headlines for dialogs

### DISCUSSION THREAD

**Discussion by dominikjannis** [Budibase/budibase#3976] [Ideas]
I got some feedback from running a small pilot project. A couple of less technophile users got a little bit unsettled by some of the hardcoded headline messages in the confirmation dialog boxes, like "Execute Query". Making these customizable would improve the overall UX of the final product by adding more context to the dialogs and be also less of an immersion breaker for non-english speaking audiences. What do you think about this?

**Comment by mjashanks** [comment]
100% agree with you - thanks for the feedback. I can't give you a timeframe for this as we have a long list of other things :)
